js/app/core/data.service.js

```diff
@@ -0,0 +1,39 @@
+((app) => {
+	const dataService = ($http, $q, constants) => {
+		const { apiRoot } = constants.urls;
```

Comment:
Using decomposition, eh? Well, aren't we fancy.

Reply:
I meant destructuring. Whatever.

Reply:
@mxsnyder eslint yelled at me, haha